.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,22 +1,36 @@
 #### What type of this PR
 
-Add one of the following kinds:
-/kind feature
-/kind bug
-/kind polish
-/kind document
+- [ ] feature
+- [ ] bug
+- [ ] polish
+- [ ] document
+
 
 #### What this PR does / why we need it:
 
+
 #### Which issue(s) this PR fixes:
 
 - Fixes #your-issue_number
 - [Erda Cloud Issue Link](paste your link here)
 
+
 #### Specified Reviewers:
 
 /assign @your-reviewer
 
+
+#### ChangeLog
+<!--
+Describe the specific changes from the user's perspective, as well as possible Breaking Change and other risks.
+-->
+
+| Language | Changelog |
+| --------- | ------------ |
+| 🇺🇸 English |              |
```

Comment:
I think it's duplicated with `#### What this PR does / why we need it:`

Reply:
agree, pr's type already tell us how this pr related to changelog.

Reply:
`What this PR does` is required, but may not required in changeLog.
If duplicated, just copy to this section.

Reply:
in this case, how you ensure that `Changelog` is really needed? (I am not talking about standard and mandatory requirement, I am talking about execution result)